src/dapp-tests/integration/tests.sh

```diff
@@ -2,15 +2,56 @@
 
 set -e
 
-export SOLC_FLAGS="--evm-version constantinople"
-export DAPPSYS_PATH="lib"
+# clean up
+trap 'kill -9 $PID_GETH && rm -rf "$TMPDIR"' EXIT
 
 error() {
     printf 1>&2 "fail: function '%s' at line %d.\n" "${FUNCNAME[1]}"  "${BASH_LINENO[0]}"
     printf 1>&2 "got: %s" "$output"
     exit 1
 }
 
+# tests some of the behaviour of
+# `dapp testnet`
+# `seth ls`
+# `seth send`
+# `seth run-tx`
+# `hevm exec`
+dapp_testnet() {
+  TMPDIR=$(mktemp -d)
+
+  dapp testnet --dir "$TMPDIR" &
```

Comment:
Note that this causes lots of processes to be left around after the script exits, because of the use of `&`.

Reply:
yeah, it needs to be killed later. I used to have it in the script but looks like it got lost